src/lib/Libattr/attr_fn_str.c

```diff
@@ -308,3 +308,29 @@ decode_jobname(attribute *patr, char *name, char *rescn, char *val)
 	}
 	return (decode_str(patr, name, rescn, val));
 }
+
+/**
+ * @brief
+ * 	set_str_extnd - set attribute value based upon another
+ *
+ *	A=B --> A is replaced with B
+ *
+ * @param[in]   attr - pointer to new attribute to be set (A)
+ * @param[in]   new  - pointer to attribute (B)
+ * @param[in]   op   - operator
+ *
+ * @return      int
+ * @retval      0       if ok
+ * @retval     >0       if error
+ *
+ */
+
+int
+set_str_extnd(struct attribute *attr, struct attribute *new, enum batch_op op)
+{
```

Comment:
I don't understand the need for this function.  The partition attribute is a string.  Why can't you use set_str().  Why do you not want someone to do a += or -=?  While those are seldom used, why block them?  Also your EDD doesn't state this.  Personally I'd rather see you use set_str().